lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb

```diff
@@ -390,6 +392,25 @@ def cmd_lcd_tabs(str, words)
     tab_complete_directory(str, words)
   end
 
+  #
+  # Create new directory on local machine
+  # 
+ def cmd_lmkdir(*args)
+   if (args.length == 0)
+     print_line("Usage: lmkdir directory")
+     return true
+   end
+
+   begin
+     ::Dir.mkdir(args[0])
```

Comment:
```suggestion
     ::FileUtils.mkdir_p(path)
```
If we switch over to using the FileUtils.mkdir_p it will recursively create the path which lines up better with the existing `mkdir` meterpreter command
Let's also have it loop over the list of args and create each directory to align with the existing command too